docs/build-protocol-info.md

```diff
@@ -0,0 +1,190 @@
+
+As an infrastructure provider, you will deal almost exclusively with signed transactions. You will,
+however, see other extrinsics within the blocks that you decode. Find more information in the
+[Substrate documentation](https://substrate.dev/docs/en/next/conceptual/node/extrinsics).
+
+Inherents contain information that is not provably true, but validators agree on based on some
+measure of reasonability. For example, a timestamp cannot be proved, but validators can agree that
+it is within some delta of their system clock. Inherents are not gossiped on the network, and only
+block authors insert them into blocks.
+
+Signed transactions contain a signature of the account that issued the transaction and stands to
+pay a fee to have the transaction included on chain. Because the value of including signed
+transactions on-chain can be recognized prior to execution, they can be gossiped on the network
+between nodes with a low risk of spam. Signed transactions fit the concept of a transaction in
+Ethereum or Bitcoin.
+
+Some transactions cannot be signed by a fee-paying account and use unsigned transactions. For
+example, when a user claims their DOTs from the Ethereum DOT indicator contract to a new DOT
+address, the new address doesn't have any fees.
+
+### Transaction Mortality
+
+Extrinsics can be mortal or immortal. The transaction payload includes a block number and block
+hash checkpoint from which a transaction is valid and a validity period (also called "era" in some
+places) that represents the number of blocks after the checkpoint for which the transaction is
+valid. If the extrinsic is not included in a block within this validity window, it will be
+discarded from the transaction queue.
+
+Setting the block checkpoint to zero, using the genesis hash, and a validity period of zero will
+make the transaction "immortal".
+
+**NOTE:** If an account is reaped and a user re-funds the account, then they could replay an
+immortal transaction. Always default to using a mortal extrinsic.
+
+### Events
+
+While extrinsics represent information from the outside world, events represent information from
+the chain. Extrinsics can trigger events. For example, the Staking pallet emits a `Reward` event
+when claiming staking rewards to tell the user how much the account was credited.
+
+### Fees
+
+Polkadot uses weight-based fees that, unlike gas, are charged _pre-dispatch._ Users can also add a
+"tip" to increase transaction priority during congested periods. See the
+[transaction fee](learn-transaction-fees) page for more info.
+
+### Encoding
+
+Parity's integration tools should allow you to deal with decoded data. If you'd like to bypass them
+and interact directly with the chain data or implement your own codec, Polkadot encodes block and
+transaction data using the [SCALE codec](https://substrate.dev/docs/en/next/conceptual/core/codec).
+
+## Smart Contracts
+
+The Polkadot Relay Chain does not expose a smart contract interface.
```

Comment:
This seems to imply that it _has_ a smart contract interface, but it's not exposed.  AFAIK it simply does not support smart contracts, right?

Reply:
Yeah, changed to make it more clear.